docs/compiler-guide.md

```diff
@@ -593,4 +598,5 @@ The F# compiler is boostrapped. That is, an existing F# compiler is used to buil
 
 ### Attribution
 
+TODO: does this link need to be kept? it just points back here
```

Comment:
Could you make the following change?

> This document is based on an original document published in 2015 by the [F# Software Foundation](http://fsharp.org). It has since been updated substantially.